### PR TITLE
fix  for support of setNames

### PR DESCRIPTION
**Pull request by jingshanglu**
Changes proposed in this pull request:
- fix support of setNames

### CODE REVIEW

**Comment by jingshanglu** [comment]
@tristaZero 
 `SET("Set", SQLStatementType.DAL)`  
 The line can't been deleted, other db use it.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/impl/MySQLDALVisitor.java [hunk 1]
```diff
@@ -297,6 +299,39 @@ public ASTNode visitSetVariable(final SetVariableContext ctx) {
...
+        SetStatement result = new SetStatement();
+        if (null != ctx.characterSetName_() || null != ctx.DEFAULT()) {
+            VariableAssignSegment characterSet = new VariableAssignSegment();
+            VariableSegment variable = new VariableSegment();
+            variable.setVariable("charset");
+            characterSet.setVariable(variable);
+            String assignValue = (null != ctx.DEFAULT()) ? ctx.DEFAULT().getText() : ctx.characterSetName_().getText();
+            characterSet.setAssignValue(assignValue);
+        }
+        if (null != ctx.collationName_()) {
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/impl/MySQLDALVisitor.java:313]: Do you think it is better to rename `collationName_`  to `collationName`?
`characterSetName_` is the same case.

Comment (jingshanglu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/impl/MySQLDALVisitor.java:313]: Ok, next pr. @tristaZero

**Pull request merged**